docs/content/latest/explore/ysql-language-features/_index.md

```diff
@@ -39,7 +39,8 @@ The following table lists the most important YSQL features which you would find
 | <span style="font-size:16px">[Query Performance](query-1-performance/)</span> | Query optimization via execution plans |
 | <span style="font-size:16px">[Stored Procedures](stored-procedures/)</span> | Support for stored procedures |
 | <span style="font-size:16px">[Triggers](triggers/)</span> | Triggers (on data modification) and event triggers (on schema changes) |
-| <span style="font-size:16px">[Table Partitions](partitions)</span> | List, range and hash partitioning of tables               |
+| <span style="font-size:16px">[Table Partitions](partitions)</span> | List, range, and hash partitioning of tables              |
+| <span style="font-size:16px">[Tablespaces](tablespaces)</span> | Definition of table spaces in YSQL |
```

Comment:
Tablespaces is a single word

Reply:
Yes, thanks, I'll fix it (the world was split automatically by the markdown editor)